### DISCUSSION TITLE
plotters only docker

### DISCUSSION THREAD

**Discussion by ghost** [guydavis/machinaris#875] [Ideas]
Is it possible to have a docker container that only controls the plotters?
no need for auto fill or anyting as long as there is the config file that we edit in regular full node and a way to start it would be awesome.

no need for mover either but it can be a cool bonus.

**Comment by guydavis** [comment]
> Is it possible to have a docker container that only controls the plotters? no need for auto fill or anyting as long as there is the config file that we edit in regular full node and a way to start it would be awesome.
> 
> no need for mover either but it can be a cool bonus.

Yes, this already exists as `mode=plotter` workers.  Please see the [wiki](https://github.com/guydavis/machinaris/wiki/Workers#plotter).

**Reply by ghost** [reply]
yes i know but what i meant was run plotter only and use flexmarmer instead of machinaris full node.
